crates/nu-command/src/filters/each.rs

```diff
@@ -132,8 +132,6 @@ with 'transpose' first."#
                             Ok(data) => Some(data.into_value(head).unwrap_or_else(|err| {
                                 Value::error(chain_error_with_input(err, is_error, span), span)
                             })),
-                            Err(ShellError::Continue { span }) => Some(Value::nothing(span)),
```

Comment:
It would be good to be clear that this also removes support for `continue` (in the PR description)

Makes sense though

Reply:
Ah yeah, I should change pr description